bot/src/web/mod.rs

```diff
@@ -40,8 +40,11 @@ struct CustomReject(anyhow::Error);
 
 impl warp::reject::Reject for CustomReject {}
 
+// TODO: Also log which endpoint caused the error
 pub(crate) fn custom_reject(error: impl Into<anyhow::Error>) -> warp::Rejection {
-    warp::reject::custom(CustomReject(error.into()))
+    let error = error.into();
+    log::error!("Endpoint error caused by: {}", error);
```

Comment:
Maybe log this in `recover` instead? Error handling is a bit messy in warp unfortunately and it would be nice to include endpoint.

Reply:
Unfortunately I haven't found anything in warp that would allow getting the endpoint path.
I will move the logging though as that is a better place for it anyway.